### ISSUE TITLE
python scripts/dream.py - ModuleNotFoundError: No module named 'ldm.simplet2i'

### ISSUE THREAD

**Issue by neilbGH** [lstein/stable-diffusion#11]
Added root folder to the search path on line 37 of dream.py to get past it. Works fine after the change.
```
sys.path.append(".")
print("* Initializing, be patient...\n")
from pytorch_lightning import logging
from ldm.simplet2i import T2I
```

**Comment by lstein** [comment]
Thank you. I’ll put that into the code.

**Comment by lstein** [comment]
Added in commit 6650b98e7c39e694e96c52cbfff381852dbf8c0d

**Comment by slix** [comment]
@lstein That commit may have caused a different problem:

```
AttributeError: module 'ntpath' has no attribute 'append'
```

**Comment by slix** [comment]
> @lstein That commit may have caused a different problem:
> 
> ```
> AttributeError: module 'ntpath' has no attribute 'append'
> ```

Probably fixed by #13
